java/explore-scene-in-flyover-ar/README.md

```diff
@@ -1,16 +1,16 @@
-# Explore scene in flyover AR
+# Explore scenes in flyover AR
 
 Use augmented reality (AR) to quickly explore a scene more naturally than you could with a touch or mouse interface.
 
-![Explore scene in flyover AR App](explore-scene-in-flyover-ar.png)
+![Image of explore scenes in flyover AR](explore-scene-in-flyover-ar.png)
 
 ## Use case
 
-You can use AR to drop into an area and visualize information, like a proposed development or a historical model of a city. You could use flyover AR to learn about a city by walking through it quickly.
+You can use AR to drop into an area and visualize information, like a proposed development or a historical model of a city. You could use flyover AR to explore a city by walking through it virtually.
 
 ## How to use the sample
 
-When you open the sample, you'll be viewing the scene from above. You can walk around, using your device as a window into the scene. Try raising and lowering your device to change altitude in the scene. Touch gesture which allow you to pan and zoom around a scene work as usual.
+When you open the sample, you'll be viewing the scene from above. You can walk around, using your device as a window into the scene. Try moving vertically to get closer to the ground. Touch gesture which allow you to pan and zoom around a scene work as usual.
 
 ## How it works
 
```

Comment:
this didn't change, but Step 6. isn't implemented. could just remove it.

Reply:
I've removed it now!